docs/en/maintain/cli.md

```diff
@@ -5,21 +5,21 @@
 
 ## NS Client
 
-Connecting to the NS Client requires specifying zk\_cluster, zk\_root\_path and role. Where zk\_cluster is the zk address, zk\_root\_path is the root path of the cluster in zk, role is the role to be started and needs to be specified as ns_client
+Connecting to the NS Client requires specifying zk\_cluster, zk\_root\_path and role. Where zk\_cluster is the zk address, zk\_root\_path is the root path of the cluster in zk and role is the role to be started, and needs to be specified as ns_client
 
 ```bash
 $ ./bin/openmldb --zk_cluster=172.27.2.52:12200 --zk_root_path=/onebox --role=ns_client
 ```
 
-### use
```

Comment:
those starting with ### (including the below) are commands, no need to capitalize them, please revert them to the lower cases